vignettes/customize-http.Rmd

```diff
@@ -12,8 +12,7 @@ vignette: >
 ```{r setup, include = FALSE}
 knitr::opts_chunk$set(
   collapse = TRUE,
-  comment = "#>",
-  eval = FALSE
+  comment = "#>"
 )
 ```
```

Comment:
You probably don't need this at all right? There's no R code running anymore

Reply:
Yeah, I guess this block can just go away entirely.